docs/sources/mimir/configure/about-versioning.md

```diff
@@ -137,7 +137,11 @@ The following features are currently deprecated and will be **removed in Mimir 2
 - Ingester
   - `-ingester.ring.readiness-check-ring-health`
 
-The following features are currently deprecated and will be **removed in Mimir 2.10**:
+The following features are currently deprecated and will be **removed in Mimir 2.11**:
 
+- Store-gateway
+  - `-blocks-storage.bucket-store.chunk-pool-min-bucket-size-bytes`
+  - `-blocks-storage.bucket-store.chunk-pool-max-bucket-size-bytes`
+  - `-blocks-storage.bucket-store.max-chunk-pool-bytes`
 - Ingester
   - `-blocks-storage.tsdb.max-tsdb-opening-concurrency-on-startup`
```

Comment:
This will be removed in 2.10 because announced in 2.8. You should just move the store-gateway config options under 2.11 removals.